x/metadata/keeper/expected_keepers.go

```diff
@@ -31,4 +33,18 @@ type AttrKeeper interface {
 // MarkerKeeper defines the attribute functionality needed by the metadata module.
 type MarkerKeeper interface {
 	GetMarkerByDenom(ctx sdk.Context, denom string) (markertypes.MarkerAccountI, error)
+	IsMarkerAccount(ctx sdk.Context, addr sdk.AccAddress) bool
+}
+
+type BankKeeper interface {
+	BlockedAddr(addr sdk.AccAddress) bool
+	MintCoins(ctx context.Context, moduleName string, amt sdk.Coins) error
+	BurnCoins(ctx context.Context, moduleName string, amt sdk.Coins) error
+	SendCoins(ctx context.Context, fromAddr, toAddr sdk.AccAddress, amt sdk.Coins) error
+	SpendableCoin(ctx context.Context, addr sdk.AccAddress, denom string) sdk.Coin
+
+	// These are methods not in the bank keeper, but that we add using our own MDBankKeeper.
+
+	DenomOwner(ctx context.Context, denom string) (sdk.AccAddress, error)
+	GetScopesForValueOwner(ctx context.Context, valueOwner sdk.AccAddress, pageReq *query.PageRequest) (types.AccMDLinks, *query.PageResponse, error)
```

Comment:
What the hell are you talking about here. They all take in a `context.Context`.

This statement is clearly not true.

> Within the BankKeeper interface, there is an inconsistency in the type of the context parameter:
> 
> Methods MintCoins, BurnCoins, and SendCoins use ctx context.Context.
> Methods SpendableCoin, DenomOwner, and GetScopesForValueOwner use ctx sdk.Context.

Further, this statement is no longer accurate:

> This could lead to confusion and potential errors, as sdk.Context is the standard context used throughout the Cosmos SDK modules.

With v0.50.x of the SDK, they've started the transition to `context.Context` for everything. The `sdk.Context` satisfies the `context.Context` interface, but most stuff now takes in a `context.Context`.

Lastly, I'm defining an interface here that needs to satisfy the bank keeper, which is outside of my control. So applying your change wouldn't work anyway.